.github/workflows/ci.yml

```diff
@@ -28,7 +28,7 @@ jobs:
       PGPASSWORD: postgres
     services:
       postgres:
-        image: postgis/postgis:14-3.3
+        image: postgis/postgis:15-3.4
```

Comment:
To be on the same page after merging:
Would there something blocking us from updating to 3.5 and isssuing warnings to <3.5?
From what I have seen, the 3.5 results seem more correct..

(lets do this in a sepaate PR, if desired)

Reply:
The feature order changed in v3.4 and MBTiles content changed([related issue here](https://trac.osgeo.org/postgis/ticket/5836)) in v3.5.
I planned to fix them one by one ( for simplicity) rather than directly upgrading to v3.5 in one single step.
But you inspired me, on second thought, I think we could upgrade to 3.5 directly since we have aligned the PostGIS versions in both `docker-compose.yml` and `ci.yml`, so the results of local and ci should be same.

Reply:
For possibly excessive caution and make this PR dedicated, let's address this in a separate PR? ^^